validphys2/src/validphys/results.py

```diff
@@ -1098,6 +1153,53 @@ def fits_chi2_table(
     return res
 
 
+@table
+def fits_nsigma_table(
+    fits_total_chi2_data,
+    fits_datasets_nsigma_table,
+    fits_groups_nsigma_table,
+    show_total: bool = False,
+):
+    """Show the nsigma of each and number of points of each dataset and experiment
+    of each fit, where experiment is a group of datasets according to the `experiment` key in
+    the PLOTTING info file, computed with the theory corresponding to the fit. Dataset that are not
+    included in some fit appear as `NaN`
```

Comment:
```suggestion
    """Show the nsigma of each and number of points of each dataset and experiment for each fit,
    computed with the theory corresponding to the fit.
    Datasets that are not included in one of the fit appear as `NaN`
```

There are probably a lot of `PLOTTING` references that we should remove...